ambry-api/src/main/java/com/github/ambry/config/ReplicationConfig.java

```diff
@@ -58,6 +59,13 @@ public class ReplicationConfig {
   @Default("false")
   public final boolean replicationEnableContinuousReplication;
 
+  /**
+   * Maximum number of iterations per remote replica group in a continuous replication cycle
+   */
+  @Config(REPLICATION_CONTINUOUS_GROUP_ITERATION_LIMIT)
+  @Default("1")
```

Comment:
Could you add explanation of default value of 1?

Reply:
Added